github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/trigger/check/BranchToCauseConverter.java

```diff
@@ -24,65 +23,76 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(BranchToCauseConverter.class);
 
     private final GitHubBranchRepository localRepo;
-    private final LoggingTaskListenerWrapper listener;
+    private final LoggingTaskListenerWrapper pollingLog;
```

Comment:
but it writes to the polling log that shows up in the gui, so this name seemed a bit clearer as to what the intended use is